### PR TITLE
Added example usages

### PR DESCRIPTION
**Pull request by egonw**
Added three: the ENM ontology, ChEMBL, and PubChem

I propose to wait until https://github.com/OBOFoundry/OBOFoundry.github.io/pull/2023 is applied/merged/finalized after which I will rebase this patch to resolve the merge conflict.

### CODE REVIEW

**Comment by matentzn** [comment]
@egonw can you resolve the conflicts please?

**Comment by egonw** [comment]
> @egonw can you resolve the conflicts please?

yes, I'll do this. later today, bc I have some work I like to finish first.

**Comment by egonw** [comment]
@matentzn: done.

File: ontology/cheminf.md [hunk 1]
```diff
@@ -25,5 +25,18 @@ publications:
   title: 'The chemical information ontology: provenance and disambiguation for chemical data on the biological semantic web'
 repository: https://github.com/semanticchemistry/semanticchemistry
 tracker: https://github.com/semanticchemistry/semanticchemistry/issues
+usages:
+- description: ChEMBL uses CHEMINF in the RDF download
+  examples:
+  - description: Physicochemical properties are represented as classes that are typed with CHEMINF classes
+    url: http://rdf.ebi.ac.uk/resource/chembl/molecule/CHEMBL6939#mw_freebase
```
Comment (matentzn) [inline ontology/cheminf.md:32]: This does not resolve for me? Certificate problems?

File: ontology/cheminf.md [hunk 2]
```diff
@@ -25,5 +25,18 @@ publications:
...
+  user: https://www.ebi.ac.uk/chembl/
+- description: ENM uses CHEMINF for chemical information terms
```
Comment (matentzn) [inline ontology/cheminf.md:34]: Unfortunately, we have decided that usages must be proven by a link to a website that lists an individual code from the ontology (like your pubchem example). An ontology re-using another does not count as usage in the sense of this metadata entry.

**Review by matentzn — CHANGES_REQUESTED** [review]

File: ontology/cheminf.md (hunk 1 above)
Comment (egonw) [reply ontology/cheminf.md:32]: ping @BZdrazil

File: ontology/cheminf.md (hunk 2 above)
Comment (egonw) [reply ontology/cheminf.md:34]: Yeah, saw it did not count; I did not read that as you cannot list it at all :)

Comment (egonw) [reply ontology/cheminf.md:34]: @matentzn, what about SPARQL endpoints? Do those count?

Comment (egonw) [reply ontology/cheminf.md]: Okay, let me know.

**Review by matentzn — APPROVED** [review]
The ChemBL usage was verified using this query:

https://api.triplydb.com/s/obBmypL2H

```
PREFIX skos: <http://www.w3.org/2004/02/skos/core#>

SELECT *
WHERE {
  ?sub ?property <http://semanticscience.org/resource/CHEMINF_000307> .
} LIMIT 10
```

on 

https://chemblmirror.rdf.bigcat-bioinformatics.org/sparql

**Comment by egonw** [comment]
@matentzn, I noticed my last commits broke the `tox` test, which I just fixed, but I did a force push for that. Sorry about that.

**Review by matentzn — APPROVED** [review]

**Comment by matentzn** [comment]
Thank you @egonw

**Pull request merged**